# HG changeset patch
# User pm
# Date 1526574120 -7200
#      Thu May 17 18:22:00 2018 +0200
# Node ID deb7e33924dcc3aaf39600b8ee0d983e21a81130
# Parent  9fc1b623a2df8c04f6e075f61fd0a877b8f0792b
Ticket mantis 0002957: dans TB17 la barre (folder pane header) est masquee

diff --git a/comm-esr60/comm/mail/base/content/messenger.xul b/comm-esr60/comm/mail/base/content/messenger.xul
--- a/comm-esr60/comm/mail/base/content/messenger.xul
+++ b/comm-esr60/comm/mail/base/content/messenger.xul
@@ -333,7 +333,7 @@
 
           <box id="messengerBox" orient="horizontal" flex="1" minheight="100" height="100" persist="height">
             <vbox id="folderPaneBox" minwidth="125" width="200" persist="width">
-              <sidebarheader id="folderPaneHeader" hidden="true" align="center"/>
+              <sidebarheader id="folderPaneHeader" align="center"/>
               <toolbar id="folderPane-toolbar"
                        class="inline-toolbar"
                        toolboxid="mail-toolbox"
